services/github/github-sponsors.service.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const gql = require('graphql-tag')
+const Joi = require('joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { NotFound } = require('..')
+const { GithubAuthV4Service } = require('./github-auth-service')
+const { documentation, transformErrors } = require('./github-helpers')
+
+const schema = Joi.object({
+  data: Joi.object({
+    repositoryOwner: Joi.object({
+      sponsorshipsAsMaintainer: Joi.object({
+        totalCount: nonNegativeInteger,
+      }),
+    }).allow(null),
+  }).required(),
+}).required()
+
+module.exports = class GithubSponsors extends GithubAuthV4Service {
+  static category = 'social'
```

Comment:
This probably should have been [`'funding'`](https://shields.io/category/funding)?

Reply:
Oops. I missed there is funding category.